.jenkins/gpuarray-branch

```diff
@@ -0,0 +1 @@
+v0.6.9
```

Comment:
This force the test to a specific tag. Do we always want that? Or do we want to test with the master of libgpuarray?